howto/README.md

```diff
@@ -45,3 +45,7 @@ Here you'll find a list of How To guides that walk you through accomplishing spe
 ## Configuring Visual Studio Code
 
 * [Debugging with daprd](./vscode-debugging-daprd)
+
+## SDKs
+
+* [Serialize state and request/response objects](./serialize)
```

Comment:
Maybe the better way is to have a terminology section at the top, saying "in this doc we'll be referring to xyz as request/response objects", because the term request/response may be confusing.  Everything is a request/response in a way.  Or maybe just "non-state" apis?

Reply:
"Serialize objects" should be enough in this page. No need to be detailed since it requires further explanation.